priv/repo/seeds.exs

```diff
@@ -106,7 +106,7 @@ Oli.Registrar.register_local_part_components(
     "janus_audio",
     "janus_video",
     "janus_slider",
-    "janus_carousel",
+    "janus_image_carousel",
```

Comment:
Renaming this part component is not sufficient to allow existing Torus instances to pick up this change.  This renaming works for seeding new databases, but existing ones will end up having the old "janus_carousel' and a new 'janus_image_carousel' both exist as part components due to how `Oli.PartComponents.register_part_component' works.  

A database migration is necessary to do this "renaming" correctly.  The migration can simply delete the 'janus_carousel' record, allowing the seed.ex change to drive the creation of a new one with the updated id.